homework/index.js

```diff
@@ -27,21 +31,136 @@
+        const nameOfContributor = contributor.login;
+        const numberOfContributor = contributor.contributions;
+        const avatarUrl = contributor.avatar_url;
+
+        const photoContributor = createAndAppend('div', rightDiv, {
+          class: 'avatar-div',
+        });
+        const infoContributor = createAndAppend('div', rightDiv, { class: 'avatar-div data' });
+
+        createAndAppend('img', photoContributor, { src: avatarUrl });
+        createAndAppend('a', infoContributor, {
+          text: nameOfContributor.toUpperCase(),
+          href: `https://github.com/${nameOfContributor}`,
+          target: '_blank',
+        });
+        createAndAppend('br', infoContributor, {});
+
+        createAndAppend('span', infoContributor, {
+          text: 'Forks: ',
+        });
+        createAndAppend('span', infoContributor, {
+          text: numberOfContributor,
+          class: 'result label',
+        });
+        createAndAppend('br', infoContributor, {});
+      });
+    } catch (error) {
+      const rightDiv = document.getElementById('cont');
+      createAndAppend('div', rightDiv, {
+        text: "there is no contributor's available",
+        class: 'alert-error',
+      });
+    }
+  }
+
+  // reload and render all data after async
+  function reloadAndRenderAllData(repository) {
+    const info = document.getElementById('info');
+    info.innerHTML = '';
+    const repositoryDescription = repository.description;
+    const fork = repository.forks;
+    const updated = repository.updated_at;
+    const dataName = repository.name;
+    const contributorsUrl = repository.contributors_url;
+
+    if (repositoryDescription == null) {
+      const noDescription = createAndAppend('li', info, {});
+      createAndAppend('span', noDescription, {
+        text: 'Description: there is no description',
+        class: '',
+      });
+    } else {
+      const liDescription = createAndAppend('li', info, {});
+      createAndAppend('span', liDescription, { text: 'Description: ' });
+      createAndAppend('span', liDescription, { text: repositoryDescription, class: 'result' });
+    }
+
+    const link = createAndAppend('li', info, {});
+    createAndAppend('span', link, { text: 'link: ' });
+    createAndAppend('a', link, {
+      text: dataName,
+      target: '_blank',
+      href: `https://github.com/HackYourFuture/${dataName}`,
+      class: 'result',
+    });
+    const liOfForks = createAndAppend('li', info, {});
+    createAndAppend('span', liOfForks, { text: 'Forks: ' });
+    createAndAppend('span', liOfForks, { text: fork, class: 'result' });
+    const liUpdateAt = createAndAppend('li', info, {});
+    createAndAppend('span', liUpdateAt, { text: 'updated_at: ' });
+    createAndAppend('span', liUpdateAt, { text: updated, class: 'result' });
+
+    // call all contributors for this selected repository
+    fetchContributorsAndRender(contributorsUrl);
+  }
+
+  function reloadAllRepositories(parenElement, allData) {
```

Comment:
I am not sure if this is the best name for this function. We are not reloading any repositories data here if I am not wrong. Instead, we are creating the dropdown and creating layout for displaying `bodyInformation`, right?